policyengine_canada/parameters/gov/provinces/ab/tax/income/benefits/acfb/working_component/reduction.yaml

```diff
@@ -0,0 +1,23 @@
+description: Alberta reduces its Child and Family Benefit working component at this rate.
+brackets:
+  - threshold:
+      2022-07-01: 0
+    rate:
+      2022-07-01: 0
+  - threshold:
+      2022-07-01: 24_467
+      2023-01-01: 25_935
+    rate:
+      2023-01-01: 0.2011
+
+# Calculated from source
+
+metadata:
+  type: marginal_rate
+  rate_unit: /1
+  threshold_unit: currency-CAD
+  label: Alberta Child and Family benefit working component reduction
```

Comment:
Also I changed this to be the working component - assume it was a typo before.

Reply:
Thsnk you!